src/main/resources/view/PatientProfile.fxml

```diff
@@ -1,20 +1,25 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import javafx.scene.control.SplitPane?>
-<?import javafx.scene.layout.AnchorPane?>
+<?import javafx.scene.control.*?>
+<?import javafx.scene.layout.*?>
 
-<AnchorPane prefHeight="400.0" prefWidth="600.0" xmlns="http://javafx.com/javafx/11.0.1" xmlns:fx="http://javafx.com/fxml/1">
-    <SplitPane dividerPositions="0.5" prefHeight="400.0" prefWidth="600.0">
+<AnchorPane prefHeight="1785.0" prefWidth="2590.0" xmlns="http://javafx.com/javafx/10.0.2-internal" xmlns:fx="http://javafx.com/fxml/1">
+    <SplitPane prefHeight="1785.0" prefWidth="2590.0">
         <items>
-          <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="160.0" prefWidth="100.0">
-              <SplitPane dividerPositions="0.5" orientation="VERTICAL" prefHeight="400.0" prefWidth="295.0">
+          <AnchorPane minHeight="0.0" minWidth="0.0" prefHeight="398.0" prefWidth="826.0">
+              <SplitPane dividerPositions="0.3868520859671302" orientation="VERTICAL" prefHeight="793.0" prefWidth="295.0">
                   <items>
-                      <AnchorPane fx:id="personalDetailsCard" minHeight="0.0" minWidth="0.0" prefHeight="100.0" prefWidth="160.0"/>
-                      <AnchorPane fx:id="medicalDetailsCard" minHeight="0.0" minWidth="0.0" prefHeight="100.0" prefWidth="160.0" />
+                      <AnchorPane fx:id="personalDetailsCard" minHeight="0.0" minWidth="0.0" prefHeight="100.0" prefWidth="160.0" />
+                      <AnchorPane fx:id="medicalDetailsCard" minHeight="0.0" minWidth="0.0" prefHeight="291.0" prefWidth="293.0" />
                   </items>
               </SplitPane>
+            <SplitPane dividerPositions="0.4066599394550959" layoutX="291.0" prefHeight="1783.0" prefWidth="2296.0">
+              <items>
+                <AnchorPane fx:id="medicalNotesPanel" minHeight="0.0" minWidth="0.0" prefHeight="793.0" prefWidth="396.0" />
+                <AnchorPane fx:id="patientChartCard" minHeight="0.0" minWidth="0.0" prefHeight="398.0" prefWidth="317.0" />
+              </items>
+            </SplitPane>
             </AnchorPane>
-          <AnchorPane fx:id="medicalNotesPanel" minHeight="0.0" minWidth="0.0" prefHeight="160.0" prefWidth="100.0" />
         </items>
     </SplitPane>
 </AnchorPane>
```

Comment:
Changes here will need to be edited later when we adjust layout of patient profile page